HostnameRedirects/Services/RewriteOptionsService.cs

```diff
@@ -72,15 +72,30 @@ public class RewriteOptionsService : IRewriteOptionsSevice, IRule {
         }
 
         private bool CheckIfIgnored(HostnameRedirectsSettings settings, string url) {
+            if(IsAdmin(url))

```

Comment:
Space after `if`.

Reply:
So this is because I wasn't running my formatter because the whole file is using brackets on the same line but my formatter drops them onto a new line.

Reply:
To be fair even other features `RobotsTxt` are using brackets on new lines...

Reply:
Happy to drop them on to new lines. Think going forward we'll be looking to use https://github.com/StyleCop/StyleCop.